.gitignore

```diff
@@ -45,3 +45,6 @@ output/
 
 myflaskapp/
 bower_components
+
+# PyCharm
+.idea/
```

Comment:
These OS- and editor-generated files should go in a global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore . Project gitignores should contain only language- and project- specific files.